camille/source/windiris.py

```diff
@@ -113,9 +113,6 @@ def windiris_internal(installation,
                           distance=None,
                           status=None):
 
-        if not os.path.isdir(root):
```

Comment:
Why is this check removed?

Reply:
It's actually moved up to windiris function itself (as in bazefetcher). I accidentally left both during rebasing

Reply:
Aha, I see it now. Thanks for the clarification.